tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -302,6 +307,16 @@ public:
       return *this;
    }
 
+    /// Return a pointer to the result, releasing its ownership and leaving this object empy. 
+    T* &Release()
```

Comment:
Why a reference to pointer?

Reply:
I added it so, while keeping in mind the way other methods have been implemented for RResultPtr, just in terms of standards